backend/src/db/migrations/20230826083159_add_user_roles_table.ts

```diff
@@ -0,0 +1,44 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_roles';
+
+const ColumnName = {
+  ID: 'id',
+  NAME: 'name',
+  KEY: 'key',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+} as const;
+
+const ROLES = [
+  {
+    id: 1,
```

Comment:
remove id, it will be added automatically
```suggestion
```

Reply:
Removed it!